client/keys/add_test.go

```diff
@@ -15,6 +15,12 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+const (
```

Comment:
I'd move these variables into the (only) function that uses them

Reply:
Updated, thanks for the review 👍